src/behaviors/shake/shake.js

```diff
@@ -0,0 +1,50 @@
+import Behavior from '../behavior.js';
+
+class ShakeText extends Behavior {
+
+  constructor(settings) {
+    settings.name = 'shake';
+    settings.description = `'cause sometimes Capslock doesn't do it!`;
+    super(settings);
+
+    this.commands.push({
+      tag: 'shake',
+      description: `I'll shake your text, cause sometimes Capslock doesn't do it!`
+    });
+  }
+
+  execute(command, message, channel, data) {
+    const parsedMessage = this.parseMessage(message, channel, data);
+	if( parsedMessage == undefined) {
+	
+	return
+	}
+	  
+    this.bot.postMessage(channel, `${parsedMessage.join('')}`, {
+      icon_emoji: ':shakehoof_beatz:',
+      thread_ts: data.thread_ts
+    });
+  }
+
+  parseMessage(message, channel, data) {
+   let splitMessage = message.replace(/^!shake/gi, '')
+   		.replace(/\:\w*\:/gi, '')
```

Comment:
Let's bring down the indentation a bit, only needs to be two spaces from the let. 👌